api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java

```diff
@@ -25,6 +26,8 @@
 
   private EptsCommonDimension eptsCommonDimension;
 
+  @Autowired private EptsCommonDimension eptsCommonDimension;
```

Comment:
We can not use @Autowired here and at the same time have its value in the public constructor that has the same annotation at the method

Reply:
That is a duplicate, there is one like that at the top, remove that line

Reply:
you will also need something like 
@Autowired 
 @Qualifier("commonAgeDimensionCohort")  
private AgeDimensionCohortInterface ageDimensionCohort;

to enable you get use to the common age dimensions

Reply:
@ningosi All implemented, thanks.